src/components/checkboxes/template.njk

```diff
@@ -31,6 +31,10 @@
     legendHtml: params.fieldset.legendHtml,
     legendHintText: params.fieldset.legendHintText,
     legendHintHtml: params.fieldset.legendHintHtml,
+    error: {
```

Comment:
I am not sure about this change… previously errorMessage mapped directly to a component, consistent with the way we handle dependent components elsewhere. I don't think it's clear that you can pass arguments consistent with the errorMessage.

I'd suggest either:

- automatically adding the error classes if an error message is passed, or
- making error a single boolean (`error: true`) and retaining the existing `errorMessage`.